lib/worker.js

```diff
@@ -143,17 +167,32 @@ Worker.prototype.run = function () {
         merge(jobAndQueueData, { result: result }),
         'Task complete'
       )
+      if (process.env.WORKER_MONITOR) {
+        var tags = this._eventTags()
+        tags.result = 'success'
+        monitor.increment('ponos.finish', tags)
+      }
```

Comment:
can you pull these `if (process.env.WORKER_MONITOR)` blocks out into some helper? It's pretty much the same block repeated multiple times, so you should be able to pull it out. Then the code isn't littered with `if`s as well